ch05.adoc

```diff
@@ -1020,3 +1019,91 @@ attributes:
 In this example the data variables **`humidity`** and **`temp`** from the <<example-h.7>> example have been replaced by the domain variable **`domain`**.
 
 ====
+
+
+[[mesh-topology-variables, Section 5.9, "Mesh Topology Variables"]]
+=== Mesh Topology Variables
+
+A __mesh topology variable__ defines the geospatial topology of cells arranged in two or three dimensions in real space but indexed by a single dimension.
+It explicitly describes the topological relationships between cells, i.e. spatial relationships which do not depend on the cell locations, via a mesh of connected nodes. 
+A mesh topology variable may provide the topology for one or more domains, defined at the nodes, edges, or faces of the mesh.
+See the <<data-model-domain-topology>> and <<data-model-cell-connectivity>> descriptions in the CF data model for more details, including on how the mesh relates to the cells of the domain.
+
+The canonical definitions of mesh topology variables and location index set variables are given externally by the UGRID conventions <<UGRID>>, but their standardized attributes, many of which are optional, are listed in <<appendix-mesh-topology-attributes>> and <<attribute-appendix>>.
+Some features of the UGRID conventions <<UGRID>> are not currently recognized by the CF conventions: mesh topology volume cells (that are used to describe fully three-dimensional unstructured mesh topologies); and the "boundary node connectivity" variable (that specifies an index variable identifying the nodes that define where boundary condtions have been provided). 
```

Comment:
I'm not sure that we need to state this, since it is a given that anything not standardised by CF is to be "ignored", and how you ignore it is up to you. No need to mention special cases. If a boundary node connectivity attribute is included then it won't affect the CF compliance of the dataset: Some client software could still choose to honour it, but some won't. This is no different in principle to including a `foo` attribute on an auxiliary coordinate variable.

Reply:
Ah, fair. That's all good, then. Marking as resolved.